projects/packages/blocks/src/class-blocks.php

```diff
@@ -49,13 +50,16 @@ public static function jetpack_register_block( $slug, $args = array() ) {
 
 		$block_type = $slug;
 
-		// If the path to block.json is passed, find the slug in the file then create a block type
-		// object to register the block.
+		// If a path is passed, find the slug in the file then create a block type object to register
+		// the block.
 		// Note: passing the path directly to register_block_type seems to loose the interactivity of
 		// the block once in the editor once it's out of focus.
 		if ( '/' === substr( $slug, 0, 1 ) ) {
-			$metadata = self::get_block_metadata_from_file( $slug );
-			$name     = self::get_block_name_from_metadata( $metadata );
+			// Test if build directory exists. Fall back to reading block.json from source directory.
```

Comment:
The build folder for blocks (`_inc/block`) is not available when running the tests from the CI jobs. I'm still looking into why this is the case. In the meantime, we fall back to reading the `block.json` file from the block source directory so that tests pass. Hopefully, this is a temporary solution. Since block assets are currently registered and enqueued manually, reading from the `block.json` file in the build or source folder doesn't matter.

This logic is implemented here so that calling `jetpack_register_block` to register a block stays as straightforward as possible.